server/controllers/activity.js

```diff
@@ -25,11 +28,20 @@ const saveActivity = (
 };
 
 const getActivities = (req, resp) => {
+  const { page } = req.params;
   const {
     user: { _id: userId }
   } = req;
+  const sanitizedPage = sanitize(page);
   let cohortIds;
   let listIds;
+  let activitiesCount;
+
+  if (!Number.isInteger(sanitizedPage) && sanitizedPage < 0) {
+    return resp.sendStatus(400);
+  }
+
+  const skip = ACTIVITIES_RES_SIZE * (sanitizedPage - 1);
```

Comment:
what is ACTIVITIES_RES_SIZE?

Reply:
number of activities to send : changed to NUMBER_OF_ACTIVITIES_TO_SEND